tools/gromacs/macros.xml

```diff
@@ -47,6 +56,14 @@
                 </param>
                 
                 <param argument="temperature" type="integer" label="Temperature /K" value="0" min="0" max="1000000" help="Temperature" />
+                <conditional name="couplinggroups">
+                   <param name="wholesystem" type="select" label="Number of groups to set for thermocoupling">
+                       <option value="true">Single coupling group (System)</option>
+                       <option value="false" selected="true">Two coupling groups (Protein and Non-Protein)</option>
+                   </param>
+                     <when value="true"/>
+                     <when value="false"/>
+                </conditional>
```

Comment:
same for this conditional

Reply:
changed this to a basic parameter with two select options.